source/overloading.tex

```diff
@@ -1810,10 +1810,10 @@
 \begin{example}
 \begin{codeblock}
 struct S {
-  std::weak_ordering operator<=>(const S&, int);  // \#1
-  std::weak_ordering operator<=>(int, const S&);  // \#2
+  friend auto operator<=>(const S&, int);  // \#1
+  friend auto operator<=>(int, const S&);  // \#2
```

Comment:
"auto" return types have extra restrictions (e.g. the function definition must be visible when calling the function). I'd suggest to revert this change.

Reply:
IIUC, the extra restrictions aren't relevant to this example. But I was just using `auto` to make the line shorter; so, if it's acceptable at this length, I don't object to putting back the explicit ordering. Done.